src/infrastructure/RepositoryFactoryHttp.ts

```diff
@@ -64,18 +64,28 @@ export class RepositoryFactoryHttp implements RepositoryFactory {
     private readonly websocketUrl: string;
     private readonly websocketInjected?: any;
     private readonly fetchApi?: any;
+    private readonly nemesisEpoch: Observable<number>;
 
     /**
      * Constructor
      * @param url the server url.
      * @param configs optional repository factory configs
      */
     constructor(url: string, configs?: RepositoryFactoryConfig) {
+        const networkRepo = this.createNetworkRepository();
         this.url = url;
         this.fetchApi = configs?.fetchApi;
-        this.networkType = configs?.networkType
-            ? observableOf(configs.networkType)
-            : this.createNetworkRepository().getNetworkType().pipe(shareReplay(1));
+        this.networkType = configs?.networkType ? observableOf(configs.networkType) : networkRepo.getNetworkType().pipe(shareReplay(1));
+        this.nemesisEpoch = configs?.nemesisEpoch
+            ? observableOf(configs.nemesisEpoch)
+            : networkRepo
+                  .getNetworkProperties()
+                  .pipe(
+                      map((property) => {
+                          return parseInt(property.network.epochAdjustment?.replace('s', '') ?? '0');
```

Comment:
Server properties may be defined in ms, seconds, hours, days, etc. Soner rather than later we would need to support multiple duration time. 

This is my implementation in java

https://github.com/nemtech/symbol-sdk-java/blob/351_configurable_epoch_time/sdk-core/src/main/java/io/nem/symbol/core/utils/FormatUtils.java#L95

Reply:
Where does this come from? I don't think this will be changed anymore since testnet is already launched.